### ISSUE TITLE
iedriver build issues

### ISSUE THREAD

**Issue by oleksiybondar** [jimevans/windows-selenium-dev-setup#1]
Hi, 

I'm not sure how to contact to you and ask a question so I'm created an issue.

So the reason why I'm trying to contact to you is that i'm trying to build iedriver from sources without any luck. I've used script from this repository and still unable to build it. I would be appreciate if you help me with this issue.

In order to avoid some extra questions I'll describe the reason why do we(by we I mean our automation team) need a custom build and what is dev environment I've set up.

So the reason is that with out web app have some conflicts with a wgxpath lib which you use in newer builds and because of this conflicts everything except xpath work just fine, and newer builds contains fixes for some issues we experienced before... and we need to have IE coverage within automation and we can go forward without xpath, also couldn't find what causing conflicts. So I have a task to create a custom build which will have all new code except wgtxpath... make a kind of a merge between new and old versions... my issue is not how to modify a code but how to build it... right now I'm trying to build master code without any modifications and no luck so far... when I try to build it using go.bat I have next output

`
Z:\UpWork\selenium>go
(in Z:/UpWork/selenium)
python Z:/UpWork/selenium/buck-out/crazy-fun/65874cbc58fab1746c2e7d15ea7c2f545fb0df38/buck.pex kill
Using watchman.
Traceback (most recent call last):
  File "Z:\UpWork\selenium\buck-out\crazy-fun\65874cbc58fab1746c2e7d15ea7c2f545fb0df38\buck.pex\.bootstrap\_pex\pex.py", line 320, in execute
  File "Z:\UpWork\selenium\buck-out\crazy-fun\65874cbc58fab1746c2e7d15ea7c2f545fb0df38\buck.pex\.bootstrap\_pex\pex.py", line 253, in _wrap_coverage
  File "Z:\UpWork\selenium\buck-out\crazy-fun\65874cbc58fab1746c2e7d15ea7c2f545fb0df38\buck.pex\.bootstrap\_pex\pex.py", line 285, in _wrap_profiling
  File "Z:\UpWork\selenium\buck-out\crazy-fun\65874cbc58fab1746c2e7d15ea7c2f545fb0df38\buck.pex\.bootstrap\_pex\pex.py", line 363, in _execute
  File "Z:\UpWork\selenium\buck-out\crazy-fun\65874cbc58fab1746c2e7d15ea7c2f545fb0df38\buck.pex\.bootstrap\_pex\pex.py", line 421, in execute_entry
  File "Z:\UpWork\selenium\buck-out\crazy-fun\65874cbc58fab1746c2e7d15ea7c2f545fb0df38\buck.pex\.bootstrap\_pex\pex.py", line 426, in execute_module
  File "C:\Pythons\python27\lib\runpy.py", line 192, in run_module
    fname, loader, pkg_name)
  File "C:\Pythons\python27\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "Z:\UpWork\selenium\buck-out\crazy-fun\65874cbc58fab1746c2e7d15ea7c2f545fb0df38\buck.pex\programs\buck.py", line 48, in <module>
  File "Z:\UpWork\selenium\buck-out\crazy-fun\65874cbc58fab1746c2e7d15ea7c2f545fb0df38\buck.pex\programs\buck.py", line 41, in main
  File "Z:\UpWork\selenium\buck-out\crazy-fun\65874cbc58fab1746c2e7d15ea7c2f545fb0df38\buck.pex\programs\buck_tool.py", line 172, in launch_buck
  File "Z:\UpWork\selenium\buck-out\crazy-fun\65874cbc58fab1746c2e7d15ea7c2f545fb0df38\buck.pex\programs\buck_tool.py", line 334, in launch_buckd
  File "C:\Pythons\python27\lib\subprocess.py", line 343, in __init__
    raise ValueError("preexec_fn is not supported on Windows "
ValueError: preexec_fn is not supported on Windows platforms
go aborted!
Buck build failed with exit code: 1
`

if I remove ..python\scripts from PATH I get'll get this output 

`
Z:\UpWork\selenium>go
(in Z:/UpWork/selenium)
python Z:/UpWork/selenium/buck-out/crazy-fun/65874cbc58fab1746c2e7d15ea7c2f545fb0df38/buck.pex kill
[-] PARSING BUCK FILES...FINISHED 3.0s [100%]
BUILT 1/308 JOBS 1.9s //java/client/src/org/openqa/selenium:beta
BUILT 2/308 JOBS 0.5s //java/client/src/org/openqa/selenium:primitives
BUILT 3/308 JOBS 0.2s //java/client/src/org/openqa/selenium/remote:http-session-id
BUILT 4/308 JOBS 0.3s //java/client/test/org/openqa/selenium/testing/drivers:browser
BUILT 5/308 JOBS 0.2s //third_party/java/cglib:cglib
BUILT 6/308 JOBS 0.2s //third_party/java/beust:jcommander
BUILT 7/308 JOBS 0.4s //java/client/src/org/openqa/selenium/remote:api
BUILT 8/308 JOBS 0.2s //third_party/java/commons:commons-exec
BUILT 9/308 JOBS 0.2s //third_party/java/commons-codec:commons-codec
BUILT 10/308 JOBS 0.1s //third_party/java/commons-fileupload:commons-fileupload
BUILT 11/308 JOBS 0.2s //third_party/java/commons:commons-lang3
BUILT 12/308 JOBS 0.2s //third_party/java/commons-io:commons-io
BUILT 13/308 JOBS 0.1s //third_party/java/commons-logging:commons-logging
BUILT 14/308 JOBS 0.2s //third_party/java/css:sac
BUILT 15/308 JOBS 0.2s //third_party/java/gson:gson
BUILT 16/308 JOBS 0.1s //third_party/java/hamcrest:hamcrest-core
BUILT 17/308 JOBS 0.3s //third_party/java/htmlunit:htmlunit-core-js
BUILT 18/308 JOBS 0.1s //third_party/java/jcip_annotations:jcip_annotations
BUILT 19/308 JOBS 0.6s //third_party/java/guava:guava
BUILT 20/308 JOBS 0.2s //third_party/java/httpcomponents:httpcore
BUILT 21/308 JOBS 0.1s //third_party/java/servlet:servlet-api
BUILT 22/308 JOBS 0.1s //third_party/java/websocket:websocket-api
BUILT 23/308 JOBS 0.3s //third_party/java/jna:core
BUILT 24/308 JOBS 0.2s //third_party/java/xml-apis:xml-apis
BUILT 25/308 JOBS 0.2s //third_party/java/yaml:snakeyaml
BUILT 26/308 JOBS 0.4s //third_party/java/jna:platform#class-abi
BUILT 27/308 JOBS 0.9s //third_party/java/guava:guava#class-abi
BUILT 28/308 JOBS 0.6s //third_party/java/jna:core#class-abi
BUILT 29/308 JOBS 0.1s //third_party/java/gson:gson#class-abi
BUILT 30/308 JOBS 0.2s //java/client/src/org/openqa/selenium:beta#class-abi
BUILT 31/308 JOBS 0.2s //java/client/src/org/openqa/selenium:primitives#class-abi
BUILT 32/308 JOBS 0.2s //third_party/java/github:github
BUILT 33/308 JOBS 0.2s //third_party/java/cssparser:cssparser
BUILT 34/308 JOBS 0.2s //third_party/java/junit:junit
BUILT 35/308 JOBS 0.2s //third_party/java/hamcrest:hamcrest-library
BUILT 36/308 JOBS 0.4s //third_party/java/httpcomponents:httpclient
BUILT 37/308 JOBS 0.4s //third_party/java/jetty:jetty-util
BUILT 38/308 JOBS 0.3s //third_party/java/jna:platform
No Android platform target specified. Using default: android-23
BUILT 39/308 JOBS 0.4s //third_party/java/xerces:xercesImpl
BUILT 40/308 JOBS 0.2s //java/client/src/org/openqa/selenium:manifest
BUILT 41/308 JOBS 0.1s //java/client/src/org/openqa/selenium:template-pom
BUILT 42/308 JOBS 0.3s //third_party/java/commons:commons-exec#class-abi
BUILT 43/308 JOBS 0.7s //java/client/src/org/openqa/selenium/logging:api
BUILT 44/308 JOBS 0.7s //java/client/src/org/openqa/selenium/support/ui:clock
BUILT 45/308 JOBS 0.5s //java/client/src/org/openqa/selenium/security:security
BUILT 46/308 JOBS 1.1s //java/client/src/org/openqa/selenium:exceptions
BUILT 47/308 JOBS 0.3s //third_party/java/httpcomponents:httpmime
BUILT 48/308 JOBS 0.2s //third_party/java/jetty:jetty-io
BUILT 49/308 JOBS 0.2s //third_party/java/htmlunit:neko-htmlunit
BUILT 50/308 JOBS 0.2s //third_party/java/xalan:serializer
BUILT 51/308 JOBS 0.1s //third_party/java/jna:jna
BUILT 52/308 JOBS 1.0s //java/client/src/org/openqa/selenium/interactions:core
BUILT 53/308 JOBS 0.2s //third_party/java/hamcrest:hamcrest-core#class-abi
BUILT 54/308 JOBS 0.3s //third_party/java/junit:junit#class-abi
BUILT 55/308 JOBS 0.1s //java/client/src/org/openqa/selenium/logging:api#class-abi
BUILT 56/308 JOBS 0.3s //java/client/src/org/openqa/selenium/support/ui:clock#class-abi
BUILT 57/308 JOBS 0.2s //third_party/java/httpcomponents:httpcore#class-abi
BUILT 58/308 JOBS 0.1s //third_party/java/commons-codec:commons-codec#class-abi
BUILT 59/308 JOBS 0.2s //java/client/src/org/openqa/selenium/remote:api#class-abi
BUILT 60/308 JOBS 0.1s //java/client/src/org/openqa/selenium/remote:http-session-id#class-abi
BUILT 61/308 JOBS 0.2s //third_party/java/commons-logging:commons-logging#class-abi
BUILT 62/308 JOBS 0.2s //java/client/src/org/openqa/selenium:exceptions#class-abi
BUILT 63/308 JOBS 0.2s //java/client/src/org/openqa/selenium/security:security#class-abi
BUILT 64/308 JOBS 0.4s //third_party/java/httpcomponents:httpclient#class-abi
BUILT 65/308 JOBS 0.2s //third_party/java/websocket:websocket-common
BUILT 66/308 JOBS 0.8s //third_party/java/xalan:xalan
BUILT 67/308 JOBS 0.2s //java/client/src/org/openqa/selenium/interactions:core#class-abi
BUILT 68/308 JOBS 0.2s //third_party/java/servlet:servlet-api#class-abi
BUILT 69/308 JOBS 0.2s //third_party/java/cglib:cglib#class-abi
BUILT 70/308 JOBS 1.5s //third_party/closure:compiler-jar
BUILT 71/308 JOBS 0.1s //javascript/atoms:bot
BUILT 72/308 JOBS 0.1s //third_party/java/jetty:jarjar-rules
BUILT 73/308 JOBS 0.1s //third_party/java/jetty:jetty-jmx
BUILT 74/308 JOBS 0.2s //third_party/java/jetty:jetty-http
BUILT 75/308 JOBS 0.1s //third_party/java/jetty:jetty-continuation
BUILT 76/308 JOBS 0.1s //javascript/atoms:errors
BUILT 77/308 JOBS 0.2s //third_party/java/commons-fileupload:commons-fileupload#class-abi
BUILT 78/308 JOBS 0.1s //third_party/java/plexus:plexus-classworlds
BUILT 79/308 JOBS 0.1s //third_party/java/plexus:plexus-component-annotations
BUILT 80/308 JOBS 0.2s //third_party/java/annotation:jsr250-api
BUILT 81/308 JOBS 0.2s //third_party/java/ant:ant-launcher
BUILT 82/308 JOBS 0.2s //third_party/java/asm:asm
BUILT 83/308 JOBS 0.2s //third_party/java/plexus:plexus-utils
BUILT 84/308 JOBS 0.1s //third_party/java/inject:javax.inject
BUILT 85/308 JOBS 0.2s //third_party/java/sisu:org.eclipse.sisu.inject
BUILT 86/308 JOBS 0.9s //java/client/src/org/openqa/selenium/logging:logging
BUILT 87/308 JOBS 1.3s //java/client/src/org/openqa/selenium:platform
BUILT 88/308 JOBS 0.4s //java/client/src/org/openqa/selenium/interactions:exceptions
BUILT 89/308 JOBS 0.1s //third_party/java/websocket:websocket-client
BUILT 90/308 JOBS 1.9s //java/client/test/org/openqa/selenium/testing:helpers
BUILT 91/308 JOBS 2.5s //third_party/closure:compiler
BUILT 92/308 JOBS 0.1s //javascript/remote:style
BUILT 93/308 JOBS 0.1s //javascript/remote:hub
BUILT 94/308 JOBS 15.6s //third_party/closure:closure
BUILT 95/308 JOBS 0.3s //third_party/java/jetty:jetty-server
BUILT 96/308 JOBS 0.2s //third_party/java/jetty:jetty-servlets
BUILT 97/308 JOBS 0.1s //cpp/prebuilt:noblur64
Defaulting to fallback for buck-out\gen\javascript\firefox-driver\nsIResponseHandler.xpt
BUILT 98/308 JOBS 0.1s //javascript/firefox-driver:webdriver_prefs
Defaulting to fallback for buck-out\gen\javascript\firefox-driver\nsICommandProcessor.xpt
Defaulting to fallback for buck-out\gen\javascript\firefox-driver\wdIMouse.xpt
Defaulting to fallback for buck-out\gen\javascript\firefox-driver\wdIStatus.xpt
Defaulting to fallback for buck-out\gen\javascript\firefox-driver\nsIHttpServer.xpt
Defaulting to fallback for buck-out\gen\javascript\firefox-driver\wdIModifierKeys.xpt
BUILT 99/308 JOBS 0.1s //cpp/prebuilt:noblur32
Defaulting to fallback for buck-out\gen\javascript\firefox-driver\wdICoordinate.xpt
BUILT 100/308 JOBS 0.1s //third_party/java/beust:jcommander#class-abi
BUILT 101/308 JOBS 0.1s //third_party/java/jcip_annotations:jcip_annotations#class-abi
BUILT 102/308 JOBS 0.3s //third_party/java/yaml:snakeyaml#class-abi
BUILT 103/308 JOBS 0.2s //third_party/java/asm:asm-tree
BUILT 104/308 JOBS 0.2s //third_party/java/maven:maven-model
BUILT 105/308 JOBS 0.2s //third_party/java/maven:maven-artifact
BUILT 106/308 JOBS 0.8s //third_party/java/ant:ant
BUILT 107/308 JOBS 0.3s //third_party/java/enterprise:cdi-api
BUILT 108/308 JOBS 0.1s //javascript/firefox-driver:response-handler-xpt
BUILT 109/308 JOBS 0.2s //java/client/src/org/openqa/selenium/logging:logging#class-abi
BUILT 110/308 JOBS 0.1s //javascript/firefox-driver:command-processor-xpt
BUILT 111/308 JOBS 0.1s //javascript/firefox-driver:httpd-js
BUILT 112/308 JOBS 0.1s //javascript/firefox-driver:mouse-xpt
BUILT 113/308 JOBS 0.1s //javascript/firefox-driver:status-xpt
BUILT 114/308 JOBS 0.2s //javascript/firefox-driver:httpd-xpt
BUILT 115/308 JOBS 0.3s //javascript/firefox-driver:content
BUILT 116/308 JOBS 0.3s //javascript/firefox-driver:modifier-keys-xpt
BUILT 117/308 JOBS 0.1s //javascript/firefox-driver:coordinates-xpt
BUILT 118/308 JOBS 0.4s //third_party/java/xalan:serializer#class-abi
BUILT 119/308 JOBS 0.8s //third_party/java/xalan:xalan#class-abi
BUILT 120/308 JOBS 0.2s //third_party/java/commons-io:commons-io#class-abi
BUILT 121/308 JOBS 1.0s //third_party/java/htmlunit:htmlunit#class-abi
BUILT 122/308 JOBS 0.3s //third_party/java/commons:commons-lang3#class-abi
BUILT 123/308 JOBS 0.3s //third_party/java/htmlunit:htmlunit-core-js#class-abi
BUILT 124/308 JOBS 0.1s //third_party/java/selenium:htmlunit-driver#class-abi
BUILT 125/308 JOBS 0.2s //third_party/java/cssparser:cssparser#class-abi
BUILT 126/308 JOBS 0.3s //third_party/java/jetty:jetty-util#class-abi
BUILT 127/308 JOBS 0.1s //third_party/java/phantomjs-driver:phantomjs-driver#class-abi
BUILT 128/308 JOBS 0.1s //third_party/java/websocket:websocket-client#class-abi
BUILT 129/308 JOBS 0.2s //third_party/java/websocket:websocket-api#class-abi
BUILT 130/308 JOBS 0.2s //third_party/java/xml-apis:xml-apis#class-abi
BUILT 131/308 JOBS 0.2s //third_party/java/jetty:jetty-io#class-abi
BUILT 132/308 JOBS 0.1s //third_party/java/htmlunit:neko-htmlunit#class-abi
BUILT 133/308 JOBS 0.4s //third_party/java/xerces:xercesImpl#class-abi
BUILT 134/308 JOBS 0.3s //third_party/java/websocket:websocket-common#class-abi
BUILT 135/308 JOBS 0.1s //third_party/java/css:sac#class-abi
BUILT 136/308 JOBS 0.2s //third_party/java/httpcomponents:httpmime#class-abi
BUILT 137/308 JOBS 0.2s //java/client/src/org/openqa/selenium:platform#class-abi
BUILT 138/308 JOBS 0.2s //java/client/src/org/openqa/selenium/interactions:exceptions#class-abi
BUILT 139/308 JOBS 0.2s //java/client/test/org/openqa/selenium/testing:helpers#class-abi
BUILT 140/308 JOBS 0.1s //javascript/firefox-driver:evaluate.js
BUILT 141/308 JOBS 0.1s //javascript/firefox-driver:error
BUILT 142/308 JOBS 0.1s //javascript/firefox-driver:timer.js
BUILT 143/308 JOBS 0.8s //third_party/java/htmlunit:htmlunit
BUILT 144/308 JOBS 0.1s //javascript/atoms:color
BUILT 145/308 JOBS 0.2s //javascript/firefox-driver:response
BUILT 146/308 JOBS 0.1s //javascript/atoms:useragent
BUILT 147/308 JOBS 0.2s //javascript/atoms:json
BUILT 148/308 JOBS 0.2s //third_party/js/wgxpath:wgxpath
BUILT 149/308 JOBS 0.1s //javascript/firefox-driver:prefs
BUILT 150/308 JOBS 0.2s //javascript/webdriver:http
BUILT 151/308 JOBS 0.2s //javascript/webdriver:logging
BUILT 152/308 JOBS 0.1s //javascript/webdriver:key
BUILT 153/308 JOBS 0.2s //java/server/src/org/openqa/selenium/remote/server:style
BUILT 154/308 JOBS 0.1s //java/server/src/org/openqa/selenium/remote/server:hub
BUILT 155/308 JOBS 0.1s //java/client/src/org/openqa/selenium/firefox:prefs
BUILT 156/308 JOBS 0.1s //third_party/java/jetty:jetty-security
BUILT 157/308 JOBS 0.1s //java/client/src/org/openqa/selenium/firefox:amd64
BUILT 158/308 JOBS 0.2s //java/client/src/org/openqa/selenium/firefox:i386
BUILT 159/308 JOBS 0.2s //third_party/java/asm:asm-commons
BUILT 160/308 JOBS 0.4s //third_party/java/sisu:org.eclipse.sisu.plexus
BUILT 161/308 JOBS 0.7s //third_party/java/github:github#class-abi
BUILT 162/308 JOBS 0.2s //java/client/test/org/openqa/selenium/testing/drivers:browser#class-abi
BUILT 163/308 JOBS 1.1s //java/client/src/org/openqa/selenium/io:io
BUILT 164/308 JOBS 0.1s //javascript/firefox-driver:files
BUILT 165/308 JOBS 1.4s //java/client/src/org/openqa/selenium/net:net
BUILT 166/308 JOBS 0.3s //third_party/firebug:firebug
BUILT 167/308 JOBS 0.1s //javascript/firefox-driver:timer
BUILT 168/308 JOBS 0.1s //javascript/firefox-driver:moz
BUILT 169/308 JOBS 0.1s //javascript/atoms:html5
BUILT 170/308 JOBS 10.1s //java/client/src/org/openqa/selenium:core
BUILT 171/308 JOBS 0.1s //javascript/atoms:domcore
BUILT 172/308 JOBS 0.1s //javascript/atoms:xpath
BUILT 173/308 JOBS 14.6s //javascript/remote:client
BUILT 174/308 JOBS 5.7s //java/server/src/org/openqa/grid/selenium:generate-build-info
BUILT 175/308 JOBS 0.2s //third_party/java/hamcrest:hamcrest-library#class-abi
BUILT 176/308 JOBS 0.2s //third_party/java/jetty:jetty-servlet
BUILT 177/308 JOBS 0.1s //third_party/java/maven:maven-plugin-api
BUILT 178/308 JOBS 0.2s //java/client/src/org/openqa/selenium/io:io#class-abi
BUILT 179/308 JOBS 0.5s //java/client/test/org/openqa/selenium/testing/drivers:firebug
BUILT 180/308 JOBS 0.2s //java/client/src/org/openqa/selenium/net:net#class-abi
BUILT 181/308 JOBS 0.1s //javascript/firefox-driver:logging
BUILT 182/308 JOBS 0.1s //javascript/firefox-driver:firefox-utils
BUILT 183/308 JOBS 0.1s //java/client/src/org/openqa/selenium:core#class-abi
BUILT 184/308 JOBS 0.1s //javascript/webdriver/atoms:attribute-lib
BUILT 185/308 JOBS 0.2s //javascript/firefox-driver:screenshooter
BUILT 186/308 JOBS 0.1s //javascript/atoms:dom
BUILT 187/308 JOBS 0.1s //java/server/src/org/openqa/selenium/remote/server:client
BUILT 188/308 JOBS 0.8s //java/server/src/org/openqa/grid/selenium:manifest
BUILT 189/308 JOBS 0.2s //third_party/java/pantsbuild:jarjar
STILL BUILDING 189/308 JOBS 59.2s //javascript/firefox-driver:session
STILL BUILDING 189/308 JOBS 1m 0.2s //javascript/firefox-driver:bad-cert-listener
STILL BUILDING 189/308 JOBS 1m 24.2s //javascript/webdriver/atoms:getAttribute
STILL BUILDING 189/308 JOBS 1m 26.0s //javascript/atoms/fragments:is-displayed
BUILT 190/308 JOBS 24.5s //javascript/firefox-driver:bad-cert-listener
BUILT 191/308 JOBS 25.3s //javascript/firefox-driver:session
BUILT 192/308 JOBS 0.1s //javascript/firefox-driver:proxy
BUILT 193/308 JOBS 0.1s //javascript/firefox-driver:web-loading-listener
BUILT 194/308 JOBS 3.9s //java/client/src/org/openqa/selenium/os:os
BUILT 195/308 JOBS 25.3s //javascript/webdriver/atoms:getAttribute
BUILT 196/308 JOBS 1.2s //javascript/atoms:locators
BUILT 197/308 JOBS 0.2s //javascript/atoms:events
BUILT 198/308 JOBS 27.1s //javascript/atoms/fragments:is-displayed
BUILT 199/308 JOBS 2.5s //java/client/src/org/openqa/selenium/interactions:interactions
BUILT 200/308 JOBS 0.1s //javascript/firefox-driver:utils
BUILT 201/308 JOBS 0.1s //java/client/src/org/openqa/selenium/os:os#class-abi
BUILT 202/308 JOBS 0.2s //java/client/src/org/openqa/selenium/remote:get-attribute
BUILT 203/308 JOBS 0.2s //javascript/atoms:devices
BUILT 204/308 JOBS 0.2s //javascript/atoms:window
BUILT 205/308 JOBS 0.1s //java/client/src/org/openqa/selenium/remote:is-displayed
BUILT 206/308 JOBS 0.1s //java/client/src/org/openqa/selenium/interactions:interactions#class-abi
BUILT 207/308 JOBS 1.5s //third_party/java/pantsbuild:jarjar-links
STILL BUILDING 207/308 JOBS 1m 54.0s //javascript/firefox-driver:web-element-cache
STILL BUILDING 207/308 JOBS 1m 53.8s //javascript/firefox-driver:modifier-keys
BUILT 208/308 JOBS 0.1s //javascript/firefox-driver:modals
BUILT 209/308 JOBS 0.0s //javascript/firefox-driver:events
BUILT 210/308 JOBS 0.0s //javascript/firefox-driver:preconditions
BUILT 211/308 JOBS 0.1s //javascript/firefox-driver:request
BUILT 212/308 JOBS 0.1s //javascript/atoms:action
BUILT 213/308 JOBS 28.7s //javascript/firefox-driver:modifier-keys
BUILT 214/308 JOBS 29.0s //javascript/firefox-driver:web-element-cache
BUILT 215/308 JOBS 0.2s //java/client/src/org/openqa/selenium:selenium
BUILT 216/308 JOBS 1.5s //third_party/java/jetty:jetty-jmx-do-repack
BUILT 217/308 JOBS 1.0s //third_party/java/jetty:jetty-servlet-do-repack
BUILT 218/308 JOBS 1.8s //third_party/java/jetty:jetty-server-do-repack
BUILT 219/308 JOBS 1.2s //third_party/java/jetty:jetty-security-do-repack
BUILT 220/308 JOBS 1.0s //third_party/java/jetty:jetty-http-do-repack
BUILT 221/308 JOBS 1.0s //third_party/java/jetty:jetty-continuation-do-repack
BUILT 222/308 JOBS 1.3s //third_party/java/jetty:jetty-io-do-repack
BUILT 223/308 JOBS 0.8s //third_party/java/jetty:jetty-servlets-do-repack
BUILT 224/308 JOBS 1.2s //third_party/java/jetty:jetty-util-do-repack
STILL BUILDING 224/308 JOBS 2m 28.4s //javascript/firefox-driver:session-store
STILL BUILDING 224/308 JOBS 2m 28.4s //javascript/firefox-driver:prompt-service
BUILT 225/308 JOBS 24.2s //javascript/firefox-driver:prompt-service
BUILT 226/308 JOBS 24.2s //javascript/firefox-driver:session-store
BUILT 227/308 JOBS 0.1s //javascript/firefox-driver:dispatcher
BUILT 228/308 JOBS 0.1s //javascript/webdriver/atoms:atoms-lib
BUILT 229/308 JOBS 0.1s //javascript/firefox-driver:modules
BUILT 230/308 JOBS 2.8s //java/client/src/org/openqa/selenium/support/events:events
BUILT 231/308 JOBS 1.3s //java/client/src/org/openqa/selenium/remote:capabilities
BUILT 232/308 JOBS 4.8s //java/client/src/org/openqa/selenium/support/ui:elements
BUILT 233/308 JOBS 8.7s //javascript/firefox-driver:synthetic-mouse
BUILT 234/308 JOBS 2.0s //java/client/src/org/openqa/selenium/support/ui:components
BUILT 235/308 JOBS 0.1s //third_party/java/jetty:jetty-jmx-repacked
BUILT 236/308 JOBS 0.1s //third_party/java/jetty:jetty-servlet-repacked
BUILT 237/308 JOBS 0.2s //third_party/java/jetty:jetty-server-repacked
BUILT 238/308 JOBS 0.1s //third_party/java/jetty:jetty-http-repacked
BUILT 239/308 JOBS 0.1s //third_party/java/jetty:jetty-security-repacked
BUILT 240/308 JOBS 0.1s //third_party/java/jetty:jetty-continuation-repacked
BUILT 241/308 JOBS 0.1s //third_party/java/jetty:jetty-io-repacked
BUILT 242/308 JOBS 0.1s //third_party/java/jetty:jetty-util-repacked
BUILT 243/308 JOBS 0.1s //third_party/java/jetty:jetty-servlets-repacked
BUILT 244/308 JOBS 0.1s //javascript/firefox-driver:webelement
BUILT 245/308 JOBS 1.1s //java/client/test/org/openqa/selenium/testing:annotations
BUILT 246/308 JOBS 0.3s //javascript/firefox-driver:js-resources
BUILT 247/308 JOBS 0.2s //java/client/src/org/openqa/selenium/support/events:events#class-abi
BUILT 248/308 JOBS 0.1s //java/client/src/org/openqa/selenium/support/ui:elements#class-abi
BUILT 249/308 JOBS 0.3s //java/client/src/org/openqa/selenium/remote:capabilities#class-abi
BUILT 250/308 JOBS 0.3s //java/client/src/org/openqa/selenium/support/ui:components#class-abi
BUILT 251/308 JOBS 0.7s //third_party/java/jetty:bundle-jars
BUILT 252/308 JOBS 0.2s //java/client/test/org/openqa/selenium/testing:annotations#class-abi
java.nio.file.FileSystemException: Z:\UpWork\selenium\buck-out\bin\java\client\src\org\openqa\selenium\remote\lib__remote-lib__classes\org\openqa\selenium\remote\getAttribute.js -> Z:\UpWork\selenium\buck-out\gen\java\client\src\org\openqa\selenium\remote\getAttribute.js: The parameter is incorrect.

        at sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:86)
        at sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:97)
        at sun.nio.fs.WindowsFileSystemProvider.createLink(WindowsFileSystemProvider.java:608)
        at java.nio.file.Files.createLink(Files.java:1086)
        at com.facebook.buck.io.ProjectFilesystem.createSymLink(ProjectFilesystem.java:950)
        at com.facebook.buck.step.fs.AbstractSymlinkFileStep.execute(AbstractSymlinkFileStep.java:79)
        at com.facebook.buck.step.fs.SymlinkFileStep.execute(SymlinkFileStep.java:25)
        at com.facebook.buck.jvm.java.CopyResourcesStep.execute(CopyResourcesStep.java:74)
        at com.facebook.buck.step.DefaultStepRunner.runStepForBuildTarget(DefaultStepRunner.java:47)
        at com.facebook.buck.rules.CachingBuildEngine.executeCommandsNowThatDepsAreBuilt(CachingBuildEngine.java:1550)
        at com.facebook.buck.rules.CachingBuildEngine.lambda$null$2(CachingBuildEngine.java:356)
        at com.facebook.buck.util.concurrent.WeightedListeningExecutorService.lambda$submitWithSemaphore$0(WeightedListeningExecutorService.java:81)
        at com.google.common.util.concurrent.AbstractTransformFuture$AsyncTransformFuture.doTransform(AbstractTransformFuture.java:211)
        at com.google.common.util.concurrent.AbstractTransformFuture$AsyncTransformFuture.doTransform(AbstractTransformFuture.java:200)
        at com.google.common.util.concurrent.AbstractTransformFuture.run(AbstractTransformFuture.java:130)
        at com.google.common.util.concurrent.MoreExecutors$5$1.run(MoreExecutors.java:988)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:748)
BUILT 253/308 JOBS 0.1s //javascript/firefox-driver:firefoxdriver
BUILT 255/308 JOBS 1.2s //java/client/src/org/openqa/selenium/support:page-factory
BUILD FAILED: //java/client/src/org/openqa/selenium/remote:remote-lib failed with exit code 1:
copy_resources
[-] BUILDING...FINISHED 3m 3.2s [100%]
Ignoring non-fatal error!  The stack trace is below:
com.facebook.buck.util.HumanReadableException: Unable to write trace file: java.nio.file.FileSystemException: Z:\UpWork\selenium\buck-out\log\build.trace -> Z:\UpWork\selenium\buck-out\log\2017-05-03_12h02m28s_buildcommand_d64c9551-1447-4153-b1dc-5db56da26119\build.2017-05-03.15-02-28.d64c9551-1447-4153-b1dc-5db56da26119.trace: The parameter is incorrect.

        at com.facebook.buck.event.listener.ChromeTraceBuildListener.outputTrace(ChromeTraceBuildListener.java:250)
        at com.facebook.buck.cli.Main.flushEventListeners(Main.java:1433)
        at com.facebook.buck.cli.Main.runMainWithExitCode(Main.java:1415)
        at com.facebook.buck.cli.Main.runMainThenExit(Main.java:744)
        at com.facebook.buck.cli.Main.main(Main.java:1932)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at com.facebook.buck.cli.bootstrapper.ClassLoaderBootstrapper.main(ClassLoaderBootstrapper.java:62)
Caused by: java.nio.file.FileSystemException: Z:\UpWork\selenium\buck-out\log\build.trace -> Z:\UpWork\selenium\buck-out\log\2017-05-03_12h02m28s_buildcommand_d64c9551-1447-4153-b1dc-5db56da26119\build.2017-05-03.15-02-28.d64c9551-1447-4153-b1dc-5db56da26119.trace: The parameter is incorrect.

        at sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:86)
        at sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:97)
        at sun.nio.fs.WindowsFileSystemProvider.createLink(WindowsFileSystemProvider.java:608)
        at java.nio.file.Files.createLink(Files.java:1086)
        at com.facebook.buck.io.ProjectFilesystem.createSymLink(ProjectFilesystem.java:950)
        at com.facebook.buck.event.listener.ChromeTraceBuildListener.outputTrace(ChromeTraceBuildListener.java:243)
        ... 9 more
Not using buckd because watchman isn't installed.
go aborted!
Buck build failed with exit code: 1
stdout:
`

so basically I'm almost there, but still I'm miss something.

my runtime env is:
Virtual Machine:
    VCPU: Intel(R) Core(TM) i5-4278U CPU @ 2.60GHz X 2
    VRAM: 2Gb
OS Name:                   Microsoft Windows 10 Pro
OS Version:                10.0.15063 N/A Build 15063

I can and will provide you any additional info you need

I wouldn't bothering you if you could figure out what's wrong by my own, 
thanx in advance

Oleksiy Bondar
extra contact info:
skype: celenia_obo
mail: bondaroleksiyandriyovich@gmail.com

**Comment by oleksiybondar** [comment]
just ignore this message, there ware some permissions issue from host machine, despite that it was 777, moving from shared folder to c drive fixed the issue
